config/filesystems.php

```diff
@@ -61,8 +62,8 @@
             'secret' => env('AWS_SECRET_ACCESS_KEY'),
             'region' => env('AWS_S3_REGION', 'us-east-1'),
             'bucket' => env('AWS_S3_BUCKET'),
-            'base_url' => 'https://s3-external-1.amazonaws.com',
-            'public_url' => env('AWS_S3_URL'),
+            'url' => env('AWS_S3_URL'),
+            'endpoint' => env('AWS_ENDPOINT'),
```

Comment:
Should I add `AWS_ENDPOINT` to the `.env.example` and would it be `https://s3-external-1.amazonaws.com`? Not sure if I matched these up right.

Reply:
Can you point me to the docs on this bit? I can see if the matching makes sense to me!

Reply:
Ok, I found these 2 PRs related to the change:

<https://github.com/laravel/laravel/pull/5267>
<https://github.com/laravel/laravel/pull/5276>

Also randomly amusing is how the PRs relate and one ends in `67` versus `76` 😆

Reply:
So from what I gather they're technically both the same, just that it should have originally been `endpoint` but for backwards compatibility they're keeping both 🤔 So I think the env should be `AWS_S3_URL` for both?

Reply:
Oh no I think I'm more confused now - are we planning to use `AWS_S3_URL` for both `url` and `endpoint`?

Reply:
Ok digging further Laravel never really had a `base_url` property here. It was added in #25 and it doesn't actually look like it's used anywhere. So I think it's safe to keep both `url` and `endpoint` and have them both use `AWS_S3_URL` which is an actual env var we do use. Thoughts?

Reply:
That sounds good to me!!